[Excel] (Custom functions) Web request article edits (#1107)
OfficeDev/office-js-docs-pr: docs/excel/custom-functions-web-reqs.md

```js
/**
 * Gets the star count for a given Github organization or user and repository.
 * @customfunction
 * @param userName string name of organization or user.
 * @param repoName string name of the repository.
 * @return number of stars.
 */

async function getStarCount(userName: string, repoName: string) {
  //You can change this URL to any web request you want to work with.

  const url = "https://api.github.com/repos/" + userName + "/" + repoName;

  let xhttp = new XMLHttpRequest();

  return new Promise(function(resolve, reject) {
    xhttp.onreadystatechange = function() {
      if (xhttp.readyState !== 4) return;

      if (xhttp.status == 200) {
        resolve(JSON.parse(xhttp.responseText).watchers_count);
      } else {
        reject({
          status: xhttp.status,

          statusText: xhttp.statusText
        });
      }
    };

    xhttp.open("GET", url, true);

    xhttp.send();
  });
}
```

### Fetch example

> It may be preferable to use the `Fetch` API over XHR for simplicity, as it avoids nested callbacks.

Author: ```suggestion
Using `Fetch` avoids nested callbacks and may be preferable in some cases to XHR. 
```
